sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+
+SRC_URI="https://github.com/sigma-star/mtd-utils/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~mips ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+S="${WORKDIR}/${PN}-v${PV}-0ef86b1"
+
+DESCRIPTION="MTD userspace tools (NFTL, JFFS2, NAND, FTL, UBI)"
+HOMEPAGE="http://git.infradead.org/?p=mtd-utils.git;a=summary"
```

Comment:
Since you have fixing to do, could you move these to correct skel positions? I would've done that after merging but then I came across the "no such directory" error...
```
EAPI=7                                                                                              
                                                                                                    
DESCRIPTION=""                                                                                      
HOMEPAGE=""                                                                                         
SRC_URI=""                                                                                          
                                                                                                    
LICENSE=""                                                                                          
SLOT="0"                                                                                            
KEYWORDS="~amd64 ~x86"                                                                              
IUSE=""                                                                                             
                                                                                                    
DEPEND=""                                                                                           
RDEPEND="${DEPEND}"                                                                                 
BDEPEND=""           
```

Reply:
Done.\
Thank you for your patience and review